### DISCUSSION TITLE
FE启动时一直停留在UNKNOWN状态，无法获取群集角色

### DISCUSSION THREAD

**Discussion by TheThirdReich77** [apache/doris#34879] [A - General / Q&A]
我在升级FE（2.0.5版本至2.0.9）时，对FE节点（非master）进行重启操作，发现FE在启动后短暂时间内能获取到FOLLOWER的角色，但过了一会就自己变成了UNKNOWN角色，导致后面的升级操作出现群集问题。
日志如下：
[FE-follower.log](https://github.com/apache/doris/files/15315543/FE-follower.log)
[FE-master.log](https://github.com/apache/doris/files/15315544/FE-master.log)

**Comment by ixzc** [comment]
How many fe's do you have? you can refer to this: https://doris.apache.org/zh-CN/docs/2.0/admin-manual/maint-monitor/metadata-operation#%E5%85%B6%E4%BB%96%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98

**Reply by TheThirdReich77** [reply]
3 fe、3 be，官方的文档已经看过了，不是所有fe节点都没启动的情况，当时是所有节点的状态都是alive的，我开始重启第一个follower fe节点就出现这种情况。

**Reply by TheThirdReich77** [reply]
另外从show frontends观察fe的状态，发现这个fe follower的状态一阵是alive的，但偶尔会出现"Sockect closed by peer"的错误，非常不稳定，同时元数据日志同步又正常进行
